pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -1,23 +1,23 @@
-{ stdenv, fetchurl, fetchgit, buildPythonPackage, python, pythonOlder,
+{ stdenv, fetchurl, fetchgit, fetchpatch, buildPythonPackage, python, pythonOlder,
   cudaSupport ? false, cudatoolkit ? null, cudnn ? null, nccl ? null, magma ? null,
-  mklSupport ? false, mkl ? null,
+  mklSupport ? false, mkl ? null, openblas ? null,
```

Comment:
We've now stabilized all of the blas and lapack provider infra in master, which means you can now rebase this and:

1. Delete the `mklSupport`, `mkl`, and `openblas` arguments
2. Add a `blas` argument, which is the virtual pkg that will be consistently instantiated with one provider across the stack.
3. When you need to figure out what it is or switch on it for doing things like adding MKL flags, you can use `blas.implementation`, which will be a string like `mkl` or `openblas`:

```
nix-repl> blas.implementation
"mkl"
```
```